src/functions-and-arrays.js

```diff
@@ -78,8 +202,35 @@ const wordsCount = [
   'matter'
 ];
 
-function howManyTimes() {}
+function howManyTimes(wordsCount) {
+  if (wordsCount.length == ""){
+    return 0; 
+  }
+  let nuevaCaja = [];
+  let numeroRep = 0;
+
+  for(o=0;o<wordsCount.length;o++){
+
+    
+
+    wordsCount.forEach(function() {
```

Comment:
Para este problema es necesario solo uno de los dos, for o forEach, con un contador.